specifications/M1/tx.dataspace.topology.md

```diff
@@ -0,0 +1,89 @@
+- A **Credential Issuer** issues Verifiable Credentials (VC) used by participant agents to allow access to assets and
+  verify usage control.
+
+## 1.1. Scope
+
+This specification does not cover dataspace registration systems managed by a Dataspace Authority (operating company) as
+the participant registration and discovery process is out-of-scope.
+
+## 1.2. Identities
+
+The DSP specifications are based on the concept that all participants have a stable identifier (ID). This ID is
+typically a number assigned to the participant business entity. In this scheme, participant agent identities may be
+ephemeral since all operations such as signing `contract agreements`are associated with the participant identity.
+
+This specification will also make use of DIDs, which can be employed to cryptographically verify a participant identity.
+These are related as follows:
+
+```
+ID  ------ Can resolve to -----> DID
+ ^                                |
+ |                                |
+ |----------Associated with--------                               
+```
+
+A DID may also be used as an ID, but it is recommended to distinguish the two to allow for DID rotation. For example, a
+participant may opt to change its hosting environment, resulting in a change to its DID such as the URL associated with
+its DID in the case of `did:web` or its DID method. Since its Identifier remains stable, existing signed contracts will
+not be impacted.
+
+# 2. Systems
+
+## 2.1. Registration System
+
+The registration system is responsible for participant registration, onboarding, and management.
+
+### 2.1.1. Registration Process
+
+When a participant is onboarded, it will be assigned an ID and either a **secret token/code** or **Membership VC** that
+will allow the participant to bootstrap its systems. If provided a secret token, the participant will be able to
+exchange it for a Membership VC with a Credential Issuer.
+
+## 2.2. Participant Agent Systems
+
+Participants will run one or more agent systems that interact in the dataspace. These systems may offer data catalogs,
+perform data transfers or provide application functionality.
+
+A participant may run the following identity-related agents. Note that this is a logical description and may not
+represent an actual deployment topology.
+
+### 2.2.1 Security Token Service (STS).
+
+The STS creates self-issued authorization tokens that contain identity claims used by participant agents under the
+control of the same participant.
+
+### 2.2.2 Verifiable Credential Service (VCS)
+
+The VCS manages [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/). This includes read and write endpoints
+for Verifiable Presentations (VPs) and Verifiable Credentials (VCs).
+
+### 2.2.3 DID Service (DIDS).
+
+The DCS Creates, signs and publishes DID documents.
+
+## 2.3. Issuer Service (IS)
+
+The Issuer Service is run by trust anchor and manages the lifecycle of Verifiable Credentials in a dataspace. A
```

Comment:
If we name the Issuer Service shouldnt we also name the Verification Service in some Kind ?

Reply:
I changed it to Credential Service - CS- since "VCS" is too close to the abbreviation "VCs" (Verifiable Credentials). 

I think this is a "credential service" since it manages credentials, but it does not verify them. Verification is done by something other than the CS. We can discuss.